CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ and this project adheres to
 - [#4926](https://github.com/firecracker-microvm/firecracker/pull/4926): Replace
   underlying implementation for seccompiler from in house one in favor of
   `libseccomp` which produces smaller and more optimized BPF code.
+- [#5021](https://github.com/firecracker-microvm/firecracker/pull/5021) If a
+  balloon device is inflated post UFFD-backed snapshot restore, Firecracker now
+  causes `remove` UFFD messages to be sent to the UFFD handler.
```

Comment:
Now I'm thinking it should go to the "fixed" section and we can also add that those "remove" messages were previously missed (it isn't that we're adding them as an extra feature).

Reply:
oops, yeah, I meant to put it into "fixed"

Reply:
fixed (lol)